src/interfaces.ts

```diff
@@ -27,4 +27,4 @@ export type ConvertOptions = {
  */
 export type ConvertFunction = (asyncapi: AsyncAPIDocument, options: ConvertOptions) => AsyncAPIDocument;
 export type ConvertOpenAPIFunction = (openapi: OpenAPIDocument, options: OpenAPIToAsyncAPIOptions) => AsyncAPIDocument;
-
+export type ConvertPostmanFunction = (postman: any) => AsyncAPIDocument;
```

Comment:
Make sure you add options here in similar terms as the OpenAPI converter options.

The `persepctive` is needed because you can both describe the requests for the server or the client.

Reply:
Done @jonaslagoni